grml-debootstrap.8.txt

```diff
@@ -226,6 +226,13 @@ Options and environment variables
     Delete grml-debootstrap configuration files (/etc/debootstrap/*) from installed
     system. Useful for reproducible builds or if you don't want to leak information.
 
+*--sshcopyauth*::
+
+    Use locally available _$HOME/.ssh/authorized_keys_ to authorise root login on the target system.
+    This installs _$HOME/.ssh/authorized_keys_ as _/root/.ssh/authorized_keys_on
```

Comment:
There is a white space missing: `s|_/root/.ssh/authorized_keys_on|_/root/.ssh/authorized_keys_ on|`